### DISCUSSION TITLE
About the bug of tool hash

### DISCUSSION THREAD

**Discussion by dubiao** [he3-app/he3-feedback#163] [Bug Report]
### Version
hash 1.0.5

### Environment
Mac v1.3.2

### Steps to reproduce
input anything

### What is expected?
output all hash values

### What is actually happening?
nothing happens

**Comment by lyzhang1999** [comment]
感谢您的反馈，已经将问题反馈给产品部门，您可以关注该 issues：https://github.com/he3-app/he3-feedback/issues/165

**Comment by congcongnalian** [comment]
非常高兴您能提出反馈！该问题已经修复，点击最新链接查看：https://t.he3app.com?d8c1
![20230512105458_rec_](https://github.com/he3-app/he3-feedback/assets/68501045/05a7edba-f846-4003-b780-008113e5f313)
